src/Misc/PerformanceViewer/performanceViewerCollectionStrategies.ts

```diff
@@ -136,10 +151,17 @@ export class PerfCollectionStrategy {
      * @returns the initializer for the draw calls strategy
      */
     public static DrawCallsStrategy(): PerfStrategyInitialization {
-        return () => {
+        return (scene) => {

```

Comment:
For some weird reason I seem to be getting a value of 0 for this strategy all the time :(. Going to see if i can figure out why

Reply:
Oh wow I figured out why HAHA, had to do with me calling capture in onBeforeRender but the sceneInstrumentation clears this value onBeforeRender and it gets priority since it is added first! I now save the number of draw calls at the end of onAfterRender and return this value when needed!